src/widgets/NewAlexandriaWidget.vue

```diff
@@ -0,0 +1,135 @@
+<script>
+  /*
+  This widget contains customizations for Explore Homer that handle
+  special-case queries tothe New Alexandria Commentary API.
+
+  In the future, we may want to take this type of customization
+  into consideration when implementing hooks / extensibility points
+  within `scaife-widgets`.
+
+  We may also find that the Vue Composition API
+  (https://composition-api.vuejs.org/) lends itself to these types of
+  customizations.
+  */
+  import gql from 'graphql-tag';
+
+  import WIDGETS_NS, {
+    NewAlexandriaWidget,
+  } from '@scaife-viewer/scaife-widgets';
+
+  export default {
+    extends: NewAlexandriaWidget,
+    name: 'NewAlexandriaWidget',
+    scaifeConfig: {
+      displayName: 'New Alexandria Commentary',
+    },
+    data: () => ({
+      passageTextParts: [],
+      healedPassage: '',
+    }),
+    computed: {
+      originalPassage() {
+        // the passage retrieved from the store
+        return this.$store.getters[`${WIDGETS_NS}/passage`];
+      },
+      passage() {
+        // overrides `NewAlexandriaWidget.computed.passage`
+        // so that only the healedPassage is passed to NAC
+        // via the `passage` watcher
+        return this.healedPassage;
+      },
+    },
+    watch: {
+      originalPassage: {
+        immediate: true,
+        handler() {
+          // special-case handling for the Folios version of
+          // the Iliad
+          if (
+            this.originalPassage.version ===
+            'urn:cts:greekLit:tlg0012.tlg001.msA-folios:'
```

Comment:
Just a minor preference here but not a blocker.  I have found that I like to put calculated conditions like this as computed properties to make it easier to troubleshoot things using Vue DevTools.

```js
    needsHealing() {
       return this.originalPassage.version === 'urn:cts:greekLit:tlg0012.tlg001.msA-folios:';
    },
```

```js
    if (this.needsHealing) {
      this.healFolioURN();
    } else {
```

Reply:
Thanks; have implemented the computed prop.